health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -7,356 +7,318 @@ import { Card } from "@egovernments/digit-ui-components";
+          <div key={`header_${ind}`}>
+            <Card className="marginBottom">
+              <HeaderComp title={bHierarchy[ind + 2]} /> {/* Wrap each bHierarchy item with Card */}
+              {parent_group?.[item]?.map((item1, idx) =>
+                Array.isArray(parents?.[item1]) && boundaryStatus?.[ind + 2] && (idx === 0 || idx === 1) ? (
+                  <SubBoundaryView key={`${item1}_${idx}`} title={item1} arr={parents?.[item1]} />
+                ) : Array.isArray(parents?.[item1]) && !boundaryStatus?.[ind + 2] ? (
+                  <SubBoundaryView key={`${item1}_${idx}`} title={item1} arr={parents?.[item1]} />
+                ) : null
+              ) || null}
+              {boundaryStatus?.[ind + 2] && parent_group[item]?.length > 2 ? (
+                <div onClick={() => handleViewMore(ind + 2)} className="view-more">
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)